### DISCUSSION TITLE
Dapatkah langsung menggunakan file migration ketika  ada perubahan database yang digunakan

### DISCUSSION THREAD

**Discussion by sispan** [uasoft-indonesia/badaso#205] [Q&A] [answered]
file migration yang terbentuk dari database manager, dapatkah digunakan langsung dengan "php artisan migrate" jika suatu saat kita akan ubah database yang hendak dipakai

**Comment by rizkiheryandi** [comment, marked as answer]
Yes, of course you can :)
